CONTRIBUTING.rst

```diff
@@ -10,14 +10,12 @@ Pontoon source code is available via GitHub:
 https://github.com/mozilla/pontoon
 
 
-Bugs
-====
-
-All bugs are tracked in `<https://bugzilla.mozilla.org/>`_.
+Issues
+======
 
-Write up a new bug:
+All bugs and ideas are tracked in `GitHub <https://github.com/mozilla/pontoon/issues>`_.
```

Comment:
You've been pretty consistent in nomenclature towards issues, this one still says bug. Nit.

```suggestion
All issues and ideas are tracked in `GitHub <https://github.com/mozilla/pontoon/issues>`_.
```